### DISCUSSION TITLE
Contributing Translating Open-Source Projects

### DISCUSSION THREAD

**Discussion by lightmagic1** [facebook/docusaurus#4097] [Q&A]
I'm looking to contribute with some open source projects translating their documentation to my main language.

In package.json i found that they're running `"@docusaurus/core": "2.0.0-alpha.63"`.

I would like to ask you guys:

1) What is the best approach to update docusaurus into a version with supported localization (i18n)?

2) Are there any migration guide into localization support? Something like detailed file structures, etc?

Thank you!

**Comment by slorber** [comment]
Hi,

It's already merged and in alpha 70, but the "official" release will be the next alpha 71 release.

You can use the `@canary` npm dist tag to use it right now, and here's the doc: https://v2.docusaurus.io/docs/next/i18n/introduction
